app/models/users.rb

```diff
@@ -0,0 +1,14 @@
+class User < ActiveRecord::Base
+  has_many :tweets
+  has_secure_password
+
+  validates_presence_of :username
```

Comment:
Hi Luke,
Thanks for your observations. There are somethings that were initially the
way you suggested but while testing the application via shotgun, some
things were not really working so I had to like tweak it a bit. For
instance the "if session[:user_id] when I took it out the test failed but
when I added it the test passed and some others also.
Be that as it may, I will still like to through it with you for better
understanding.
Once again, thank you so much, I really appreciate it.

Johnson.

On Wed, Aug 24, 2016 at 12:47 PM, Luke Ghenco <notifications@github.com>
wrote:

> In app/models/users.rb
> <https://github.com/learn-co-students/sinatra-fwitter-group-project-v-000/pull/193#discussion_r76094240>
> :
>
> > @@ -0,0 +1,14 @@
> > +class User < ActiveRecord::Base
> > +  has_many :tweets
> > +  has_secure_password
> > +
> > +  validates_presence_of :username
>
> good use of active record method here.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/learn-co-students/sinatra-fwitter-group-project-v-000/pull/193/files/59824eb76c4de11b38eb901fca404959d820769f#r76094240>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AOoAYzg23XxvZyuADsPuMf0u2UFcGehjks5qjHWigaJpZM4JpdWl>
> .
>